src/cache.c

```diff
@@ -286,6 +305,12 @@ cache_t *cache_create(int size_bits)
         mpool_create(cache_size * sizeof(lfu_entry_t), sizeof(lfu_entry_t));
 #endif
     cache->capacity = cache_size;
+#if RV32_HAS(JIT)
+    cache->jitcode = (uint8_t *) mmap(NULL, CODE_CACHE_SIZE,
```

Comment:
The casting from `void *` to `uint8_t *` is not necessary in C programming language.